tools/tasks.py

```diff
@@ -178,7 +178,7 @@ def execute(self, fp):
             p = subprocess.Popen(
                 self.command,
                 bufsize=-1,
-                stdin=subprocess.PIPE,
+                stdin=subprocess.DEVNULL,
```

Comment:
I was surprised to see this is not a change already made in cbcollect_info if it's actually required, and I am also not sure quite why this change?

Surely sgcollect_info was not completely broken on Windows up until this point?

The commit message isn't really indicative of the change being made. There's nothing Windows or CI specific about this change, and we're not closing stdin.

Will this change impact interactive password prompts in future? I guess maybe not since that's not stdin, and it's instead just a prompt from the program.